src/main/java/org/sonarsource/sonarlint/ls/notebooks/VersionedOpenNotebook.java

```diff
@@ -117,7 +114,6 @@ public Set<String> getCellUris() {
   }
 
   public Optional<URI> getCellUri(int lineNumber) {
-    indexCellsByLineNumber();
```

Comment:
This is in order to avoid redundant re-indexing. This method will always be called after the `toCellIssue()` one, which triggers re-indexing itself.